### DISCUSSION TITLE
Including first and last sections during chapter export

### DISCUSSION THREAD

**Discussion by niccottrell** [mifi/lossless-cut#1639] [Q&A]
Hi! I'm using the extract chapters feature on a video I made. I have a chapter marker at 00:00 and towards the end, but the tool marks the first and last chapters with a trashcan icon. But, I want to export these along with all the other chapters. Is there a way to _not_ skip them?

<img width="1085" alt="Screenshot 2023-06-29 at 17 47 54" src="https://github.com/mifi/lossless-cut/assets/850094/903e34d6-abd4-491c-8ad5-12a1e1cfa9a2">

**Comment by IFONDERWR** [comment]
same problem

**Reply by mifi** [reply]
Hi, sorry I missed this. is it possible to share a file that shows the problem?
I;m not sure what you're trying to do exactly. are you loading chapters from a video as segments, and it gives you segments 1 to 10, however you have an 11th chapter which goes from 00:00 to the end of the video which doesn't show up as a segment in losslesscut?

**Reply by niccottrell** [reply]
Hi Mifi! It's not specific to a file - I think it's an assumption in the UI that the first cutpoint always indicates lead in time to discard. In my case, I want to keep the whole content of the video just break it into smaller pieces.

So, I want to break up a video into chunks but including the 00:00 to the first cutpoint. In my screenshot above, you can see that the first and last sections are marked with a trashcan and don't get exported. Maybe I've misunderstood the UI. Am I missing the UI workflow to undo the default trashcan choice.

**Reply by mifi** [reply]
Hmm, I cannot reproduce the problem with my file that has chapters in it.

1. open the file that has embedded chapters
2. get question "This file has embedded chapters. Do you want to import the chapters as cut-segments?", answer "Import chapters"
3. now I see this:

<img width="863" alt="Screenshot 2024-12-18 at 16 10 47" src="https://github.com/user-attachments/assets/76956770-f4ac-420a-ac4d-851a6d786051" />

<img width="191" alt="Screenshot 2024-12-18 at 16 10 40" src="https://github.com/user-attachments/assets/94524ffb-3c1d-43ef-9f38-35180ee16ced" />

for reference here are the chapters from MacOS quicklook for the input file:

<img width="284" alt="Screenshot 2024-12-18 at 16 10 15" src="https://github.com/user-attachments/assets/f54b2d35-13cc-40b4-9239-a44adebac0a2" />

so is it possible to share a file that shows this problem?
